backend/data/src/main/kotlin/io/tolgee/service/notification/NotificationService.kt

```diff
@@ -33,15 +35,19 @@ class NotificationService(
       .toInt()
 
   @Transactional
-  fun save(notification: Notification) {
-    emailNotificationsService.sendEmailNotification(notification)
-    notificationRepository.save(notification)
-    applicationEventPublisher.publishEvent(
-      OnNotificationsChangedForUser(
-        notification.user.id,
-        notification,
-      ),
-    )
+  fun notify(notification: Notification) {
```

Comment:
Please check that notifications are not created for deleted users. (and test it)

You can create notifications for disabled users, but don't send them e-mails.

Reply:
Will be done in another PR.